File: sdk/javascript-closure-angular/API/Client/EmailTemplate.js
goog.provide('API.Client.EmailTemplate');

/**
 * @record
 */
API.Client.EmailTemplate = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.EmailTemplate.prototype.subject;

/**
 * @type {!string}
 * @export
 */
API.Client.EmailTemplate.prototype.body;

/**
 * @type {!string}
 * @export
 */
API.Client.EmailTemplate.prototype.replyEmail;

/**
 * @type {!boolean}
 * @export
 */
API.Client.EmailTemplate.prototype.sendMessageType;

/**
 * @type {!string}
 * @export
 */
API.Client.EmailTemplate.prototype.emailFormatType;

File: sdk/dart/lib/model/email_template.dart
part of swagger.api;


@Entity()
class EmailTemplate {
  
  @Property(name: 'Subject')
  String subject = null;
  

  @Property(name: 'Body')
  String body = null;
  

  @Property(name: 'ReplyEmail')
  String replyEmail = null;
  

  @Property(name: 'SendMessageType')
  bool sendMessageType = null;
  

  @Property(name: 'EmailFormatType')
  String emailFormatType = null;
  
  EmailTemplate();

  @override
  String toString()  {
    return 'EmailTemplate[subject=$subject, body=$body, replyEmail=$replyEmail, sendMessageType=$sendMessageType, emailFormatType=$emailFormatType, ]';
  }

}

